test/math.jl

```diff
@@ -387,12 +387,20 @@ j33 = besselj(3,3.)
 @test besselj(-3,-3) == j33
 @test besselj(-3,3) == -j33
 @test besselj(3,-3) == -j33
+@test besselj(3,3f0) ≈ j33
```

Comment:
okay so if the answer is no even to the rounded version, then not worth worrying about